APIs/IPMXAPI.raml

```diff
@@ -0,0 +1,91 @@
+#%RAML 1.0
+
+title: IPMX Device Control
+baseUri: http://example.api.com/x-nmos/ipmx/{version}
+version: v1.0
+mediaType: application/json
+
```

Comment:
```suggestion

documentation:
  - title: Overview
    content: |
      The EDID API is exposed by NMOS Devices as a standard control interface for management of inter-device media essences.
  - title: Endpoint Discovery
    content: |
      The EDID API URL for an NMOS Device is advertised in the list of controls for that device in the Node API using type: "urn:x-nmos:edid:sr-ctrl/{version}". This information is populated into the NMOS registry via the standard registration mechanism, so it is discoverable through the Query API (see NMOS Discovery and Registration specification).
  - title: API Structure
    content: |
      The API presents (snippet from IS-05 for now) two distinct interfaces. A "bulk" interface for controlling multiple senders/receivers, and a "single" interface to control single devices.
      
```

Reply:
This looks about right to me.  We can discuss further and commit on Thursday call.